File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/plagiarized/L6/04/Kasus2.kt
package IR

import java.util.Scanner

/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
fun main() {
    val input = Scanner(System.`in`)
    // Enter radius of the cylinder
    print("Enter the radius and length of a cylinder: ")
    val r = input.nextDouble()
    val l = input.nextDouble()
    println("The area is ${r * r * Math.PI}")
    println("The volume of the cylinder is ${r * r * Math.PI * l}")
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L6/04/Kasus2.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

console.log("Enter the radius and length of a cylinder: ");

rl.on('line', (input) => {
  const [r, l] = input.split(' ').map(Number);
  console.log("The area is " + r*r*Math.PI);
  console.log("The volume of the cylinder is " + r*r*Math.PI*l);
  rl.close();
});

